samtranslator/model/sam_resources.py

```diff
@@ -764,6 +771,9 @@ class SamApi(SamResourceMacro):
         "Stage": ApiGatewayStage.resource_type,
         "Deployment": ApiGatewayDeployment.resource_type,
         "DomainName": ApiGatewayDomainName.resource_type,
+        "UsagePlan": ApiGatewayUsagePlan.resource_type,
+        "UsagePlanKey": ApiGatewayUsagePlanKey.resource_type,
```

Comment:
`Ref` does nothing for UsagePlanKeys, we shouldn't include it here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-usageplankey.html

Reply:
`Ref` for `ApiGatewayUsagePlanKey` gives the physical id which has the mapping `ApiKeyPhysicalId:UsagePlanPhysicalId`. I think it was a miss in the docs

Reply:
Sounds good, this looks good to me then